### DISCUSSION TITLE
Omniauth ABAC

### DISCUSSION THREAD

**Discussion by kuwv** [omniauth/omniauth#1112] [Q&A]
I'm researching ABAC capability with Omniauth. Is this already supported or is it possible to extend this by creating a plugin?

Essentially, I'm looking for ability to use the existing Authentication piece and only replace (or extend) the existing RBAC for the Authorization component.

**Comment by BobbyMcWho** [comment]
Omniauth strategies are pretty flexible, there are tons of custom strategies that pull different data from their respective IDP, it's just about taking the data the IDP sends back and associating that with whatever user model you're using
